src/App.js

```diff
@@ -1,16 +1,23 @@
+// import { render } from '@testing-library/react';
 import './App.css';
+import React, {useState} from 'react';
+
+// const multiplier = 0;
+// const chickenRate = 0;
+// const total = 0; 
```

Comment:
Why are these comments present? Are they needed?

Reply:
They will be removed in the next branch.

Reply:
I don't like that we aren't keeping the history clean, can we just remove them now?

Reply:
No

Reply:
ive done it. is it to your satisfaction?

Reply:
Perfect thanks!